tools/hangar/src/cli.test.ts

```diff
@@ -144,15 +151,53 @@ test.each(validWingFiles)(
 test.each(validWingFiles)(
   "wing test %s (--target sim)",
   async (wingFile) => {
-    const command = ["test"];
+    const args = ["test"];
     const testDir = path.join(tmpDir, `${wingFile}_sim`);
     fs.mkdirpSync(testDir);
 
-    await runWingCommand(command, path.join(validTestDir, wingFile), testDir);
+    await runWingCommand(
+      testDir,
+      path.join(validTestDir, wingFile),
+      args,
+      true
+    );
 
     // TODO snapshot .wsim contents
   },
   {
     timeout: 1000 * 30,
   }
 );
+
+test.each(invalidWingFiles)(
+  "wing test %s (--target sim) - invalid",
+  async (wingFile) => {
+    const args = ["test"];
+    const testDir = path.join(tmpDir, `invalid_${wingFile}_sim`);
+    fs.mkdirpSync(testDir);
+
+    const out = await runWingCommand(
+      testDir,
+      path.join(invalidTestDir, wingFile),
+      args,
+      false
+    );
+
+    const stderr = out.stderr;
+
+    const stderrSanitized = stderr
+      // Remove absolute paths
+      .replaceAll(invalidTestDir.replaceAll("\\", "/"), "<TEST_DIR>")
```

Comment:
Could we just use relative paths instead? I feel like showing relative paths to your source files in compiler error messages by default is a better user experience since it reduces the amount of unimportant information on your screen